### DISCUSSION TITLE
No data in Grafana

### DISCUSSION THREAD

**Discussion by ftt1211** [teslamate-org/teslamate#4050] [Q&A]
I've been having issues for months, and am finally trying to figure out why grafana no longer pulls data. Can anyone help me figure out this issue and how to fix? 
![image](https://github.com/teslamate-org/teslamate/assets/24555213/6864d9c5-eb3c-431a-98a4-f356348ab5ed)

**Comment by cwanja** [comment]
What is your setup? What do the Grafana logs say?

**Reply by ftt1211** [reply]
My setup on my Synology NAS used to work fine up until I upgraded to 7.2 about 8 months ago. Then grafana had issues so I reinstalled it, but I've been seeing these errors. Here is what the log says:
![image](https://github.com/teslamate-org/teslamate/assets/24555213/aacb9c15-6755-4ca1-9e6a-1ad8fe59a35a)
